inc/ticket.class.php

```diff
@@ -2776,34 +2782,71 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
                         }
                      }
 
+                     if (isset($input['with_documents'])) {
+                        // Create new links for any Documents
+                        $tomerge = $document_item->find([
+                           'items_id' => $id,
+                           'itemtype' => 'Ticket'
+                        ]);
+
+                        foreach ($tomerge as $key => $document_item2) {
+                           $document_item->deleteByCriteria([
```

Comment:
Instead of deleting the document to recreate it just after, it may be better to do not create it if we find it already attached to the ticket (i.e. `if (count > 0) continue;`).